bitnami/contour/templates/service.yaml

```diff
@@ -28,13 +28,16 @@ metadata:
     {{- include "contour.tplValue" (dict "value" .Values.envoy.service.labels "context" $) | nindent 4 }}
     {{- end }}
   annotations:
+    {{- if (ne (get .Values.envoy.service.annotations "service.beta.kubernetes.io/aws-load-balancer-type") "nlb") }}
```

Comment:
While working on #3381 and testing my changes with helm 2.14 and 2.16 I get this error:
```console
helm2 lint .

==> Linting .
[ERROR] templates/: parse error in "contour/templates/service.yaml": template: contour/templates/service.yaml:34: function "get" not defined

Error: 1 chart(s) linted, 1 chart(s) failed
```
I realized that this is a function introduced in [Sprig v3](https://github.com/Masterminds/sprig/releases/tag/v3.0.0): https://github.com/Masterminds/sprig/pull/197.

Since bitnami needs helm v2 compat, I think we should try to solve this using an available function.

Reply:
I agree with you @mkilchhofer !! Would you open a PR addressing this? thanks in advance